ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__automated-release-stable-4.16-upgrade-from-stable-4.15.yaml

```diff
@@ -0,0 +1,102 @@
+base_images:
+  ansible:
+    name: "4.15"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.15"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openstack-installer:
+    name: "4.15"
+    namespace: ocp
+    tag: openstack-installer
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.15"
```

Comment:
I think `tests-private-postupg` should use 4.16 here.

Reply:
+1

Reply:
even we don't run pre/post upgrad test in upgrade job. anyway, we need to correct this config setting

Reply:
@tomasdavidorg let us apply this change, then I'll add label lgtm for this PR.